src/common/com.smi.microservices.common/src/main/java/org/smi/common/execution/SmiShutdownHook.java

```diff
@@ -16,8 +18,11 @@ public SmiShutdownHook(IMicroserviceHost host) {
 	 */
 	@Override
 	public void run() {
-
 		if (_host != null)
-			_host.Shutdown();
+			try {
+				_host.Shutdown();
+			} catch (IOException e) {
+				e.printStackTrace();
```

Comment:
Does this get written to the log file? I don't think we always capture the stderr at the moment so this info may be lost.

Reply:
All we're doing at that point is closing the Rabbit connection - if it fails I don't think we have much to log anyway. (It wasn't logged earlier, either, just printed to stderr - I've just moved the call "up" a bit in the call stack.)